lib/src/async.dart

```diff
@@ -19,7 +19,7 @@ class _FutureHook<T> extends Hook<AsyncSnapshot<T>> {
 
   final Future<T> future;
```

Comment:
The Future (and Stream with useStream) should be nullable

Reply:
> @DevNico could you give me the rights to make edits on your PR? Thanks!

The checkbox to allow edits from maintainers is already enabled or do you mean something else?